src/foam/java/refinements.js

```diff
@@ -96,25 +100,25 @@ foam.CLASS({
   methods: [
     function createJavaPropertyInfo_(cls) {
       return foam.java.PropertyInfo.create({
-        sourceCls:        cls,
-        propName:         this.name,
-        propShortName:    this.shortName,
-        propAliases:      this.aliases,
-        propType:         this.javaType,
-        propValue:        this.javaValue,
-        propRequired:     this.required,
-        cloneProperty:    this.javaCloneProperty,
-        diffProperty:     this.javaDiffProperty,
-        jsonParser:       this.javaJSONParser,
-        queryParser:      this.javaQueryParser,
-        csvParser:        this.javaCSVParser,
-        extends:          this.javaInfoType,
+        sourceCls: cls,
```

Comment:
Why this change?

Reply:
Fixed: https://github.com/foam-framework/foam2/pull/1239